CHANGELOG.md

```diff
@@ -52,6 +52,136 @@ See: [.changelog/unreleased](.changelog/unreleased)
+* `github.com/golang/glog` bumped to v1.2.2 (from v1.2.1) [PR 2157](https://github.com/provenance-io/provenance/pull/2157).
+* `github.com/gorilla/websocket` bumped to v1.5.3 (from v1.5.1) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/klauspost/compress` bumped to v1.17.9 (from v1.17.7) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/munnerz/goautoneg` added at v0.0.0-20191010083416-a7dc8b61c822 [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/prometheus/client_golang` bumped to v1.20.1 (from v1.19.1) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/prometheus/common` bumped to v0.55.0 (from v0.52.2) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/prometheus/procfs` bumped to v0.15.1 (from v0.13.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/rs/cors` bumped to v1.11.1 (from v1.11.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `github.com/spf13/cast` bumped to v1.7.0 (from v1.6.0) [PR 2114](https://github.com/provenance-io/provenance/pull/2114).
+* `golangci-lint` bumped to v1.60.2 (from v1.54.2) [PR 2132](https://github.com/provenance-io/provenance/pull/2132).
+* `golang.org/x/crypto` bumped to v0.25.0 (from v0.23.0) [PR 2107](https://github.com/provenance-io/provenance/pull/2107).
+* `golang.org/x/crypto` bumped to v0.26.0 (from v0.25.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `golang.org/x/net` bumped to v0.27.0 (from v0.25.0) [PR 2107](https://github.com/provenance-io/provenance/pull/2107).
+* `golang.org/x/net` bumped to v0.28.0 (from v0.27.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `golang.org/x/oauth2` bumped to v0.21.0 (from v0.20.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `golang.org/x/oauth2` bumped to v0.22.0 (from v0.21.0) [PR 2157](https://github.com/provenance-io/provenance/pull/2157).
+* `golang.org/x/sync` bumped to v0.8.0 (from v0.7.0) [PR 2115](https://github.com/provenance-io/provenance/pull/2115).
+* `golang.org/x/sys` bumped to v0.22.0 (from v0.20.0) [PR 2107](https://github.com/provenance-io/provenance/pull/2107).
+* `golang.org/x/sys` bumped to v0.23.0 (from v0.22.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `golang.org/x/sys` bumped to v0.24.0 (from v0.23.0) [PR 2157](https://github.com/provenance-io/provenance/pull/2157).
+* `golang.org/x/term` bumped to v0.22.0 (from v0.20.0) [PR 2107](https://github.com/provenance-io/provenance/pull/2107).
+* `golang.org/x/term` bumped to v0.23.0 (from v0.22.0) [PR 2142](https://github.com/provenance-io/provenance/pull/2142).
+* `golang.org/x/text` bumped to v0.17.0 (from v0.16.0) [PR 2115](https://github.com/provenance-io/provenance/pull/2115).
```

Comment:
I thought the script merged these types of dependency updates into a single line... not a huge deal but there are a lot of entries in this list that would be streamlined.

Reply:
The old dependbot changelog thing would combine entries, but the new way doesn't have it yet. I'm hoping to make the `prep-release.sh` script combine them, but haven't had time for it yet.

When it creates the full release, it'll replace all the release candidate entries in the changelog with a combined single entry. So if I get the dependency entry combination done before then, our full release will end up having them combined and we won't have this long list anymore.

The tricky thing is that our new dependency changelog stuff also keeps track of libraries being added and removed, so I need to work through what to do with various combinations of added, bumped, and removed lines.

Reply:
would be super easy to manually roll up since they are all nicely sorted.